authgear-nft-indexer.yaml.example

```diff
@@ -16,3 +16,8 @@ server:
   listen_addr: 0.0.0.0:8080
 alchemy:
```

Comment:
Change the schema to the following

```yaml
alchemy:
- blockchain: ethereum
  network: "1"
  api_key: THE_API_KEY
- blockchain: ethereum
  network: "5"
  api_key: THE_API_KEY
```

This is consistent with our previous model. Implementation-wise, we only recognize ethereum:0x0@1 and ethereum:0x0@5 and can generate the Alchemy endpoint for these two.